addons/account/views/account_invoice_view.xml

```diff
@@ -645,13 +645,55 @@
 
         <menuitem action="action_invoice_tree1" id="menu_action_invoice_tree1" parent="menu_finance_receivables" sequence="1"/>
 
+        <record id="action_invoice_out_refund" model="ir.actions.act_window">
+            <field name="name">Customer Refunds</field>
```

Comment:
As per task https://www.odoo.com/web#id=32014&view_type=form&model=project.task&action=333&active_id=248&menu_id=4720
we should now use "Credit Note" instead of "Refund"...

(this comment applies to the whole diff)